From: SeongJae Park <sj@kernel.org>
Subject: samples/damon/wsse: rename to have damon_sample_ prefix
Date: Sat, 5 Jul 2025 10:49:55 -0700

Patch series "mm/damon: misc cleanups".

Yet another round of miscellaneous DAMON cleanups.


This patch (of 6):

DAMON sample module, wsse has its name 'wsse'.  It could conflict with
future modules, and not very easy to identify it by name.  Use a prefix,
"damon_sample_" for the name.

Note that this could break users if they depend on the old name.  But it
is just a sample, so no such usage is expected, or known.  Even if such
usage exists, updating it for the new name should be straightforward.

Link: https://lkml.kernel.org/r/20250705175000.56259-1-sj@kernel.org
Link: https://lkml.kernel.org/r/20250705175000.56259-2-sj@kernel.org
Signed-off-by: SeongJae Park <sj@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 samples/damon/wsse.c |    5 +++++
 1 file changed, 5 insertions(+)

--- a/samples/damon/wsse.c~samples-damon-wsse-rename-to-have-damon_sample_-prefix
+++ a/samples/damon/wsse.c
@@ -12,6 +12,11 @@
 #include <linux/kernel.h>
 #include <linux/module.h>
 
+#ifdef MODULE_PARAM_PREFIX
+#undef MODULE_PARAM_PREFIX
+#endif
+#define MODULE_PARAM_PREFIX "damon_sample_wsse."
+
 static int target_pid __read_mostly;
 module_param(target_pid, int, 0600);
 
_
